cli/README.md

```diff
@@ -0,0 +1,194 @@
+    show                show job(s) by uuid
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --cluster CLUSTER, -c CLUSTER
+                        the name of the Cook scheduler cluster to use
+  --url URL, -u URL     the url of the Cook scheduler cluster to use
+  --config CONFIG, -C CONFIG
+                        the configuration file to use
+  --retries RETRIES, -r RETRIES
+                        the number of retries to use for HTTP connections
+  --verbose, -v         be more verbose/talkative (useful for debugging)
+```
+
+All global options (`--cluster`, `--config`, etc) can be provided when using subcommands.
+
+#### `submit`
+
+You can submit one or more jobs with `submit`. A single command can be provided as a positional argument, or multiple commands can be provided, one per line, via `stdin`. Jobs can also be passed as "raw" JSON data by setting the `--raw` flag. The `submit` command returns UUIDs, one per line, for each created job.
+
+```
+$ cs submit --help
+usage: cs submit [-h] [--uuid UUID] [--name NAME] [--priority]
+                 [--max-retries MAX-RETRIES] [--max-runtime MAX-RUNTIME]
+                 [--cpus CPUS] [--mem MEM] [--raw]
+                 [command] ...
+
+positional arguments:
+  command
+  args
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --uuid UUID, -u UUID  uuid of job
+  --name NAME, -n NAME  name of job
+  --priority , -p       priority of job, between 0 and 100 (inclusive)
+  --max-retries MAX-RETRIES
+                        maximum retries for job
+  --max-runtime MAX-RUNTIME
+                        maximum runtime for job
+  --cpus CPUS           cpus to reserve for job
+  --mem MEM             memory to reserve for job
+  --raw, -r             raw job spec in json format
+```
+
+#### `wait`
+
+You can wait for jobs to complete with `wait`. Job UUIDs are passed as positional arguments.
+
+```
+$ cs wait --help
+usage: cs wait [-h] [--timeout TIMEOUT] [--interval INTERVAL] uuid [uuid ...]
+
+positional arguments:
+  uuid
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --timeout TIMEOUT, -t TIMEOUT
+                        maximum time (in seconds) to wait
+  --interval INTERVAL, -i INTERVAL
+                        time (in seconds) to wait between polling
+```
+
+#### `show`
+
+You can query jobs with `show`. Job UUIDs are passed as positional arguments. This command returns a table showing the fields and values for the queried jobs. If you use `--json`, the command returns a JSON representation of the queried jobs.
+
+```
+$ cs show --help
+usage: cs show [-h] [--json] uuid [uuid ...]
+
+positional arguments:
+  uuid
+
+optional arguments:
+  -h, --help  show this help message and exit
+  --json      show the job(s) in JSON format
+```
+
+### Examples
+
+Simple job creation:
+```shell
+$ cs submit echo 1
+ddeebd40-656c-44c4-9843-9e5e8e06ae5a
```

Comment:
This output should be more verbose and explain what happened.  Perhaps there is a command line option to only emit a UUID if desired.